### ISSUE TITLE
Can we combine the two datasets?

### ISSUE THREAD

**Issue by Redsandro** [needim/wdt-emoji-bundle#20]
We have a long string of emoji data in `emoji.min.js` and again but different in `defaults.emojiData`. Can we have just one?

**Comment by needim** [comment]
Yep, you are right. I created a new object because I wanted performance gain for the loops. (mainly for this function: wdtEmojiBundle.fillPickerPopup())

It would be nice to have just one object, maybe we can find a better way. And this bundle uses sheets instead of single images.
